design/open/unity-catalog-exporter.md

```diff
@@ -0,0 +1,31 @@
+# Unity catalog exporter
+
+## Introduction
+
+Currently, due to the limitations of Databricks Unity catalog, which supports only cloud provider direct storage
+endpoints and authentication, it's not feasible to configure it to work directly with lakeFS.  
+We wish to overcome this limitation to enable Unity catalog-backed services to read Delta Lake tables, which is the
+default table format used by Databricks, from lakeFS.
+
+---
+
+## Proposed Solution
+
+Following the [catalog exports issue](https://github.com/treeverse/lakeFS/issues/6461), the Unity catalog exporter will 
+utilize the [Delta Lake catalog exporter](./delta-catalog-exporter.md) to export an existing Delta Lake table to 
+`${storageNamespace}/_lakefs/exported/${ref}/${commitId}/${tableName}`. Following this, it will create an external table
+in an existing `catalog.schema` within the Unity catalog, using the Databricks API, the provided 
+`_lakefs_tables/<table>.yaml` definitions by the user, and specifying the location where the Delta Log was exported to.
```

Comment:
How many external tables can Unity realistically support? The documentation for Delta Share was 1k, although we managed to get to 100k until we had issues. This means that 100k isn't a good answer either. Should we also consider deregistering tables?

Reply:
The [limitations](https://docs.databricks.com/en/data-governance/unity-catalog/index.html#resource-quotas) are:
* 10_000 schemas per catalog
* 10_000 tables per schema

That means we can support 10_000 tables per branch.
Was the problem the total amount of tables repo-wise? If so, this solution allows a total of 1,000,000,000 tables per catalog (repo).